bindings/python/src/peer_info.cpp

```diff
@@ -82,7 +83,7 @@ void bind_peer_info()
         .def_readonly("download_queue_length", &peer_info::download_queue_length)
         .def_readonly("upload_queue_length", &peer_info::upload_queue_length)
         .def_readonly("failcount", &peer_info::failcount)
-        .def_readonly("downloading_piece_index", &peer_info::downloading_piece_index)
+        .add_property("downloading_piece_index", make_getter(&peer_info::downloading_piece_index, by_value()))
```

Comment:
would you mind giving the same treatments to other fields that is a piece-index or file-index too?